django_project/ford3/static/js/qualification_form.js

```diff
@@ -21,6 +66,41 @@ let addSubject = (e) => {
     let formCheck = $('.form-check');
     formCheck.addClass('form-check-inline');
 
+    // Multiple select
+    let $selectMultiple = $('.selectmultiple');
+    if ($selectMultiple.length) {
+        $selectMultiple.parent().prev().removeClass('col-md-3').addClass('col-md-12');
+        $.each($selectMultiple, function (index, selectDiv) {
```

Comment:
Could you refactor this code with functions so we have a better idea what does it do?

Reply:
And maybe add some tests with Jest.

Reply:
I don't think we need to refactor it, because this code is only specific for this form. We could do that in the future, and make it reusable.

Reply:
Make it more readable by humans then